crates/ubrn_common/src/rust_crate.rs

```diff
@@ -133,7 +133,7 @@ fn so_extension_from_target<'a>(target: &str) -> &'a str {
         "a"
     } else if target.contains("android") {
         // We're using staticlib files here. cargo ndk use .so files.
-        "a"
+        "so"
```

Comment:
I don't know if we know what the consequences are for the `ubrn build` for React Native.

Reply:
Yeah, let me undo this one for now, there's a separate effort to address this, it must have snuck in.